.github/workflows/build.yaml

```
@@ -0,0 +1,161 @@
+        (github.event_name == 'workflow_dispatch' && github.event.inputs.revision) ||
+        github.sha }}
+      PLATFORMS: ${{ (github.event_name == 'workflow_call' && inputs.platforms) ||
+        (github.event_name == 'workflow_dispatch' && github.event.inputs.platforms) ||
+        'linux,macos,windows' }}
+
+    steps:
+      - name: Initialize job summary
+        run: echo "# Build & Package Summary for ${{ matrix.os }}" >> $GITHUB_STEP_SUMMARY
+
+      - name: Skip if this platform not requested
+        if: ${{ !contains(fromJson(format('[\"{0}\"]', join(',', env.PLATFORMS))), matrix.os) }}
```

actionlint (1.7.7)

number of arguments is wrong. function "join(array<string>) -> string" takes 1 parameters but 2 arguments are given

(expression)